src/taipy/gui/_gui_cli.py

```diff
@@ -9,15 +9,26 @@
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
-from taipy.config.common._argparser import _Argparser
+from argparse import Namespace, ArgumentError
+from taipy.config._cli._cli import _CLI
 
 
 class _GuiCLI:
     """Command-line interface of GUI."""
+    _DEFAULT_ARGS = {
+        "port": "",
+        "host": "",
+        "ngrok_token": "",
+        "webapp_path": "",
+        "debug": False,
+        "no_debug": False,
+        "use_reloader": False,
+        "no_reloader": False,
+    }
 
     @classmethod
-    def _create_parser(cls):
-        gui_parser = _Argparser._add_groupparser("GUI", "Optional arguments for GUI service")
+    def create_parser(cls):
+        gui_parser = _CLI._add_subparser("taipy", help="Run application with Taipy arguments.")
```

Comment:
This I think breaks the legacy parsing mechanism.
Users would for example use `python script.py -debug` and get what they expect.
Now it seems that forces them to use `python script.py taipy -debug`.

Reply:
What I'm saying is that we need to rethink this thing and try to allow not to use the `taipy` option